dtdl2-validator/Properties/launchSettings.json

```diff
@@ -1,8 +1,31 @@
 {
   "profiles": {
-    "dtdl2-validator": {
+    "default-resolver": {
       "commandName": "Project",
-      "commandLineArgs": "..\\..\\..\\..\\test\\demodevice.json"
+      "commandLineArgs": "/f=../../../../test/demodevice.json"
+    },
+    "public-resolver": {
+      "commandName": "Project",
+      "commandLineArgs": "/f=../../../../test/demodevice.json /resolver=public"
+    },
+    "shared-resolver": {
+      "commandName": "Project",
+      "commandLineArgs": "/f=../../../../test/demodevice.json /resolver=public /modelRepoUrl=https://raw.githubusercontent.com/ridomin/registry/master/"
+    },
+    "local-resolver": {
+      "commandName": "Project",
+      "commandLineArgs": "/f=../../../../test/demodevice.json /resolver=local /baseFolder=../../../../../registry/models"
+    },
+    "none-resolver": {
+      "commandName": "Project",
+      "commandLineArgs": "/f=../../../../test/demodevice.json /resolver=none"
+    },
+    "private-resolver": {
+      "commandName": "Project",
+      "commandLineArgs": "/f=../../../../test/demodevice.json /resolver=private",
+      "environmentVariables": {
+        "StorageConnectionString": "DefaultEndpointsProtocol=https;AccountName=contosomodels;AccountKey=UxpVYHpDpwQBEtUMUAu/zj7rT1uxnXFjDqMwnWlyf11yt/LEe/lvE07NgfgfzNwnvDElx6vlPkUevXTeZl7lXA==;EndpointSuffix=core.windows.net"
```

Comment:
I know we should not commit secrets, but this is intentional for easy testing. The data in the storage account is a copy of the public models